scripts/make-release.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
See above. We can keep this as-is. No strong opinions though.

Reply:
Cuz `/bin/sh` doesn't actually support `-o pipefail` on some systems. Forcing bash in this case is portable.